### ISSUE TITLE
[Bug] Dialogs are not hidden correctly

### ISSUE THREAD

**Issue by schlagmichdoch** [schlagmichdoch/PairDrop#335]
> When used with a screen reader such as orca on linux in firefox or NVDA on windows the user experience is as if all the dialogs were shown at the same time.
> I guess not all of them are visible on the screen but instead of hiding them with CSS they are moved off screen or masked in anyway.
> Pressing tab and shift+tab reaches controls that are supposed to be hidden.
> The obvious way on how to test this is that the language selector is always reachable no matter if I press its corresponding close button or not.

_Originally posted by @pvagner in https://github.com/schlagmichdoch/PairDrop/issues/306#issuecomment-2354646022_

**Comment by schlagmichdoch** [comment]
Apparently, the dialogs are only hidden by setting the opacity to 0. This also explains some startup issues there are with slow browsers sometimes.

@pvagner Would setting it to `display: none` solve this issue for screen readers like yours?

**Comment by pvagner** [comment]
@schlagmichdoch Yes, setting ``` display: none ``` would work for me and other screen reader users I believe. Thank you for looking into it.

**Comment by schlagmichdoch** [comment]
I guess we have to use a timeout in JavaScript to add `display:none` to the dialogs after a timeout. I have it on the roadmap.
